packages/gatsby/src/query/redirects-writer.ts

```diff
@@ -31,18 +34,14 @@ const writeRedirects = async () => {
   )
 }
 
-exports.writeRedirects = writeRedirects
-
-let bootstrapFinished = false
-let oldRedirects
 const debouncedWriteRedirects = _.debounce(() => {
   // Don't write redirects again until bootstrap has finished.
   if (
     bootstrapFinished &&
     !_.isEqual(oldRedirects, store.getState().redirects)
   ) {
     writeRedirects()
-    oldRedirects = store.getState().Redirects
+    oldRedirects = store.getState().redirects
```

Comment:
I correct this code because I think it's a typo

Reply:
That's nice catch!

But there is some asterisk regarding this entire file - we actually don't use it anywhere 😱 

It seems after we moved this file to https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/bootstrap/redirects-writer.js there was some git messup and old file location stayed. What's really weird is that the file we actually use - don't even have that check